### PR TITLE
sdlada 2.5.4-1

### PR DESCRIPTION
**Pull request by mosteo**

### CODE REVIEW

**Review by Fabien-Chouteau — DISMISSED** [review]

File: index/sd/sdlada/sdlada-2.5.4-1.toml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+command = ["make", "-C", "build/gnat", "SDL_PLATFORM=windows"]
+
+[available."case(os)"]
+'linux|macos|windows' = true
+"..." = false
+
+[available."case(distribution)"]
+# There is some problem I don't readily know how to solve:
+# /usr/sbin/ld: /usr/lib/libharfbuzz.so.0: undefined reference to `fstat64@GLIBC_2.33'
+arch = false
```
Comment (onox) [inline index/sd/sdlada/sdlada-2.5.4-1.toml:43]: @mosteo Have you tried `pacman -Syu`? This version of the crate works just fine here on Arch (I did get an exception in alr when trying to upgrade sdlada 2.3 with `alr update` if I just set `arch = true`, so I had to remove the whole block).

Comment (mosteo) [reply index/sd/sdlada/sdlada-2.5.4-1.toml:43]: Perhaps our test docker for Arch is stale. I will try refreshing it. Do you have any more info on the exception?

Comment (mosteo) [reply index/sd/sdlada/sdlada-2.5.4-1.toml:43]: Nvm, I just hit the same problem and got the stack trace.

Comment (mosteo) [reply index/sd/sdlada/sdlada-2.5.4-1.toml:43]: Tracking it at https://github.com/alire-project/alire/issues/719

Comment (mosteo) [reply index/sd/sdlada/sdlada-2.5.4-1.toml:43]: The good news is that with the updated Arch image it indeed builds, so I'll enable it.

**Pull request merged**